package.json

```diff
@@ -48,7 +48,7 @@
   },
   "homepage": "https://github.com/pagopa/io-backend#readme",
   "dependencies": {
-    "@pagopa/io-spid-commons": "^3.1.1",
+    "@pagopa/io-spid-commons": "^3.2.0",
```

Comment:
can you please document SAML_ACCEPTED_CLOCK_SKEW_MS into the readme ?

Reply:
Added in baad0ae